apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -308,55 +320,42 @@ export const FileBrowser = React.memo(() => {
             message = codebridgeI18n.duplicateSupportFileError({fileName});
           }
         }
-        if (message) {
-          dialogControl?.showDialog({
-            type: DialogType.GenericAlert,
-            title: message,
-          });
-          return true;
-        } else {
-          return false;
-        }
-      },
-    [dialogControl]
-  );
```

Comment:
github merged a few things together.

The relevant change is that `checkForDuplicateFilename` will now return an error message if it finds a duplicate (or undefined if it's not a dupe) instead of showing the `GenericAlert` dialog. These error messages are now handled from within the `GenericPrompt`.